cns/middlewares/k8sSwiftV2_windows.go

```diff
@@ -40,5 +46,12 @@ func (k *K8sSWIFTv2Middleware) assignSubnetPrefixLengthFields(podIPInfo *cns.Pod
 		},
 		GatewayIPAddress: interfaceInfo.GatewayIP,
 	}
+	// assign the default route with gateway IP
```

Comment:
i feel its not added in right function. either create new function to add this or add in function where routes are configured